### PR TITLE
Blocks copy tool

### PR DESCRIPTION
**Pull request by pstibrany**
Tool for copying blocks between GCS buckets.

State of PR: tool is tested and works, but I will add metrics to it.

### CODE REVIEW

**Comment by pstibrany** [comment]
This is now ready for revew. Thanks in advance!

File: cmd/blockscopy/main.go [hunk 1]
```diff
@@ -0,0 +1,477 @@
...
+// IsCopiedToBucketMarkFilename returns whether the input filename matches the expected pattern
+// of copied markers stored in markers location.
+// Target bucket is part of the mark filename, and is returned as 3rd return value.
+func IsCopiedToBucketMarkFilename(name string) (bool, ulid.ULID, string) {
+	parts := strings.SplitN(name, "-", 3)
+	if len(parts) != 3 {
+		return false, ulid.ULID{}, ""
+	}
+
+	// Ensure the 2nd part matches the block deletion mark filename.
```
Comment (pracucci) [inline cmd/blockscopy/main.go:465]:
```suggestion
	// Ensure the 2nd part matches the block copy mark filename.
```

**Review by pracucci — APPROVED** [review]
Solid work, easy to follow. LGTM!

**Comment by pstibrany** [comment]
Thanks for review!

**Pull request merged**